### ISSUE TITLE
Twitter API now requires submission of application

### ISSUE THREAD

**Issue by brunogarciagonzalez** [learn-co-curriculum/twitter-api-todo#27]
A student brought to my attention that the Twitter API bonus lab is deprecated because, "twitter makes you submit an application to gain access to their api now (as of a few weeks ago)"

**Comment by sgharms** [comment]
@brunogarciagonzalez What do you mean by "is deprecated?" By whom? Or did the student mean this lab "ought be deprecated?"

**Comment by brunogarciagonzalez** [comment]
@sgharms Hello, I just double checked the definition of the word and I think I picked the wrong word to express the thought. My point is that twitter has changed its api and so the students cannot complete the lab as they could before. [Here](https://twitter.com/twitterapi?lang=en) is a link to the TwitterAPI twitter account, they have been implementing changes to end points and the student was saying that to gain access to the api they needed to submit an application to twitter.

Let me rephrase the original comment: A student stated that due to the Twitter api changes, the lab cannot be completed, it appears. I am opening this Issue to bring it to the Curriculum team's attention, for the team to look into these changes by twitter and change the lab accordingly. It may be that in the mean time the lab should not be deployed.

**Comment by sgharms** [comment]
Upon further review, and discussion with the instructional team, @maxwellbenton has decided the best path forward for this lab is to remove it. We'll do so in our next deployment.

We're tracking changes to Tracks (which require deployments) here: https://github.com/learn-co-curriculum/flatiron-curriculum-online-proposals/blob/master/README.md

We'll add a note here suggested that this lab be skipped and, with the next roll out, it will cease to exist :)

**Comment by sgharms** [comment]
Update! It's already been undeployed.
